src/react/src/util/styles.js

```diff
@@ -654,6 +654,78 @@ export const makeDownloadExcelButtonStyles = theme =>
         }),
     });
 
+export const makeDashboardContributionRecordStyles = theme =>
+    Object.freeze({
+        loaderStyles: Object.freeze({
+            display: 'block',
+            margin: 'auto',
+        }),
+        emptyBlockStyles: Object.freeze({
+            height: '100px',
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+        }),
+        emptyTextStyle: Object.freeze({ color: 'rgb(142 138 138 / 87%)' }),
+        buttonStyles: Object.freeze({
+            margin: '20px',
+            padding: '20px',
+            width: '100%',
+            fontWeight: 'bold',
+        }),
+        container: Object.freeze({
+            marginBottom: '25px',
+            width: '100%',
+            padding: '20px',
+        }),
+        title: Object.freeze({
+            paddingBottom: '20px',
+        }),
+        listItemStyle: Object.freeze({
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'space-between',
+        }),
+        listItemTextStyle: Object.freeze({
+            padding: 0,
+        }),
+        dividerStyle: Object.freeze({
+            height: '3px',
+            backgroundColor: COLOURS.DARK_GREY,
+        }),
+        innerDividerStyle: Object.freeze({
+            backgroundColor: COLOURS.DARK_GREY,
+        }),
+        prettyPrint: Object.freeze({
+            width: '100%',
+            maxHeight: '300px',
+            overflow: 'auto',
+        }),
+        potentialMatchesBlock: Object.freeze({
+            width: '100%',
+            marginBottom: '25px',
+        }),
+        potentialMatchesInternalBlock: Object.freeze({
+            width: '100%',
+            overflow: 'auto',
+            maxHeight: '450px',
+        }),
+        buttonsContainerStyles: Object.freeze({
+            display: 'flex',
+            flexDirection: 'row',
+            width: '100%',
+        }),
+        confirmButtonStyles: Object.freeze({
+            backgroundColor: '#b1dbb1',
+            padding: '20px',
+            '&:hover': { backgroundColor: '#799679' },
```

Comment:
Please add '#799679' to the COLOURS.js and name It some similar to MINT_GREEN.

Reply:
done